TrafficCapture/dockerSolution/src/main/docker/migrationConsole/django/console_api/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for console_api project.
+
+Generated by 'django-admin startproject' using Django 5.0.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-26h*wo1qzffhpum=bn#8d(7e8mo-w9fr6*wdy#%izy#5^85-a9'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+DEPLOYED_STAGE = os.environ.get('MIGRATION_STAGE')
+ALLOWED_HOSTS = ['migration-console', f'migration-console.migration.{DEPLOYED_STAGE}.local', 'localhost']
+
+
+# Application definition
```

Comment:
Lets try to get the minimal set of apps/middlewhere we need.  We can add them in later if we need the functionality.